openqabot/types/baseconf.py

```diff
@@ -1,17 +1,17 @@
 # Copyright SUSE LLC
 # SPDX-License-Identifier: MIT
-from abc import ABCMeta, abstractmethod, abstractstaticmethod
 from typing import Any, Dict, List, Optional
 
 from .incident import Incident
 
 
-class BaseConf(metaclass=ABCMeta):
+class BaseConf:
+    OBEY_EMBARGOED_CONF = "PUBLIC_CLOUD_RUN"
```

Comment:
no , this is pretty awful ...

Reply:
why ? please provide some practical reasoning . such comment is applicable to movie that you have watched or food which you tried . In code review it is not allow me to keep meaningful discussion . you saying "this is awful" and I am saying "no this is cool" and we are blocked ...

From my side I can say that such approach allow to avoid typos in variable name when it is used in several places and ease change of its name

Reply:
resolved